R/run_trisk.R

```diff
@@ -154,7 +158,7 @@ run_stress_test_iteration <- function(args_list) {
 }
 
 # Avoid R CMD check NOTE: "Undefined global functions or variables"
-globalVariables(c(names(formals(run_trisk)), "iter_var", "end_year"))
+globalVariables(c(names(formals(run_trisk)), "iter_var", "end_year", "market_passthrough"))
```

Comment:
you should not have to add market_passthrough here, it is included in formals(run_trisk)

Reply:
done